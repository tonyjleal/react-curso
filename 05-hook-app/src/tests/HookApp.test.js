import React from 'react';
import {shallow} from 'enzyme';
import { HookApp } from '../HookApp';
import '@testing-library/jest-dom'

describe('Component HookApp', () => { 

    test('should show <HookApp /> success', () => {
    
        const wrapper = shallow(<HookApp />);

        expect(wrapper).toMatchSnapshot();

     });

    
     test('should show Hola Mundo into element H1', () => {

        const texto = 'Hola Mundo';

        const wrapper = shallow(<HookApp />);

        const textoH1 = wrapper.find('h1').text();

        expect(textoH1).toBe(texto);

     })


 })