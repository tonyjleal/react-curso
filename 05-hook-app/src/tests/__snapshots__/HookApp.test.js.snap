// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component HookApp should show <HookApp /> success 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
